.github/workflows/publish-chromatic.yml

```diff
@@ -17,8 +18,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  chromatic_deployment:
-    name: Publish Chromatic
+  chromatic_deployment8:
```

Comment:
We need to support both storybook 6 and v8 simultaneously.

Reply:
This was removed just for testing. I will bring it back when the PR is ready for review